src/relayer/Relayer.ts

```diff
@@ -273,7 +273,11 @@ export class Relayer {
         maxBlockNumber,
         transactionHash: deposit.transactionHash,
       });
-      return;
+      // If we're in simulation mode, skip this early exit so that the user can evaluate
+      // the full simulation run.
+      if (this.config.sendingRelaysEnabled) {
```

Comment:
Should this be negated?

```suggestion
      if (!this.config.sendingRelaysEnabled) {
```